packages/providers/fal/avatar-generation/fal_avatar/models/kling.py

```
@@ -0,0 +1,466 @@
+        self.defaults = MODEL_DEFAULTS["kling_v2v_reference"]
+        self.max_duration = MAX_DURATIONS["kling_v2v_reference"]
+
+    def validate_parameters(
+        self,
+        prompt: str,
+        video_url: str,
+        duration: Optional[str] = None,
+        aspect_ratio: Optional[str] = None,
+        audio_url: Optional[str] = None,
+        face_id: Optional[str] = None,
+        **kwargs,
```

Ruff (0.14.10)

Unused method argument: `kwargs`

(ARG002)